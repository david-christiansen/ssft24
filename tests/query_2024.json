["object", {"year":["is", 2024]}]
