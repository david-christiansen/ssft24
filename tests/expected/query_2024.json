{"year": 2024, "name": "bobkonf"}
